### DISCUSSION TITLE
付款后，仍然提示即将到期，且到期日期没有变化。

### DISCUSSION THREAD

**Discussion by arpitec2000** [getlantern/lantern-forum-cn#566] [General]
专业版邮箱为：arpitec@163.com
付款凭证如下：
![Weixin Image_20240613195614](https://github.com/getlantern/lantern-forum-cn/assets/102727113/ccc927b2-a82a-4e24-a1fd-f41579776f9e)
付款时间为：2024年6月13日 17点
付款3个小时候，专业版账户信息截图如下：
![Screenshot 2024-06-13 195835](https://github.com/getlantern/lantern-forum-cn/assets/102727113/f0bf946d-7d93-4318-9500-aec6e6a677a1)

请协调处理续费结果或退款均可。

**Comment by arpitec2000** [comment]
还是不行哦，仍然显示即将过期，麻烦尽快处理下。

**Comment by MaxMKMax** [comment]
@arpitec2000 您好. 您放心, 我会帮您提交问题. 下周给您答复, 谢谢您的耐心等待.
